init_templates/MAIN.db4-set.template

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<set><title>Example Set</title>
+
+ <book id="book.a"><title>Example Book 1</title><subtitle>generated by DAPS</subtitle><bookinfo>
```

Comment:
This file is used as an example, so it would be beneficial to improve the formatting. Could you try to move the `title` and `subtitle` into the next lines and indent it?